.github/workflows/plugin_install.yml

```diff
@@ -3,63 +3,121 @@ name: Plugin Install
+          COPY --chown=1001:1001 opensearch* /opensearch/
+          RUN chmod +x /docker-host/os-ep.sh
+          RUN useradd -u 1001 -s /sbin/nologin opensearch
+          ENV PATH="/opensearch/bin:${PATH}"
+          WORKDIR /opensearch/
+          ENTRYPOINT /docker-host/os-ep.sh
+          EOF
+          docker run --name ops -d -p 9200:9200 -p 9600:9600 -i opensearch-test:latest
+
+      - name: Sleep while OpenSearch finishes starting up
+        uses: whatnick/wait-action@v0.1.2
+        with:
+          time: '30s'
+
+      - name: Check OpenSearch Running
+        run: curl -XGET https://localhost:9200 -u 'admin:admin' -k -v
+
+      - name: Get Docker Logs
+        if: always()
+        run: docker logs ops
+
+      - name: Run sanity tests
+        run: ./gradlew integTestRemote -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="opensearch" -Dhttps=true -Duser=admin -Dpassword=admin
+
+  windows-install:
+    if: ${{ false }}  # disable for now - no stable distribution for 3.0.0 yet
+
+    runs-on: windows-latest
+
+    steps:
+      - name: Enable longer filenames for Windows
+        run: git config --system core.longpaths true
+
+      - name: Checkout Plugin
+        uses: actions/checkout@v2
+
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Download OpenSearch Core
+        run: |
+          cd ..
+          Invoke-WebRequest https://artifacts.opensearch.org/snapshots/core/opensearch/3.0.0-SNAPSHOT/opensearch-min-3.0.0-SNAPSHOT-windows-x64-latest.zip -Outfile opensearch-3.0.0.zip
+          tar -xzf opensearch-3.0.0.zip 
+          del opensearch-3.0.0.zip
+          Rename-Item opensearch-* Opensearch
+
+      - name: Build Security Plugin with Gradle
+        run: |
+          .\gradlew.bat assemble
+        env:
+          _JAVA_OPTIONS: -Xmx4096M
+
+      - name: Move and rename security plugin
+        run: move build\distributions\opensearch-security-*.zip ..\opensearch-security.zip
+
+      - name: Install the plugin
+        run: |
+          cd ..
+          'y' | .\Opensearch\bin\opensearch-plugin.bat install file:\a\security\opensearch-security.zip
+          'y', 'y', 'N' | .\Opensearch\plugins\opensearch-security\tools\install_demo_configuration.bat
+
+      - name: Run OpenSearch with plugin
+        run: |
+          cd ..
+          start .\Opensearch\bin\opensearch.bat
+
+      - name: Sleep while OpenSearch finishes starting up
+        run: Start-Sleep -s 30
+
+      - name: Check OpenSearch Running
+        run: |
+          $credentialBytes = [Text.Encoding]::ASCII.GetBytes("admin:admin")
+          $encodedCredentials = [Convert]::ToBase64String($credentialBytes)
+          $baseCredentials = "Basic $encodedCredentials"
+          $Headers = @{ Authorization = $baseCredentials }
+          Invoke-WebRequest -SkipCertificateCheck -Uri 'https://localhost:9200' -Headers $Headers
```

Comment:
Save some code and rewrite as `Invoke-WebRequest -SkipCertificateCheck -Uri 'https://admin:admin@localhost:9200'` , worked when I tried it locally.  More details on this format [1]

[1] https://serverfault.com/questions/371907/can-you-pass-user-pass-for-http-basic-authentication-in-url-parameters